pages/common/docker-swarm.md

```diff
@@ -0,0 +1,32 @@
+# docker swarm
+
+> A container orchestration tool.
+> More information: <https://docs.docker.com/engine/swarm/>.
+
+- Initialize a swarm cluster:
+
+`docker swarm init`
+
+- Get join token for master or worker:
+
+`docker swarm join-token {{type}}`
+
+- Join new node to cluster:
+
+`docker swarm join --token {{token}} {{manager_node_url:2377}}`
+
+- Leave the cluster:
```

Comment:
The doc says `issue the following command from worker2 itself`:
```suggestion
- Remove a worker from the swarm (run inside the worker node):
```
Should this command be run in the worker itself?

Reply:
yes.
even inside the manager node can run it but sometimes need an extra `--force` option to run on the manager node